cmd/k8s-bigip-ctlr/main.go

```diff
@@ -419,7 +446,8 @@ func main() {
 		log.Fatalf("error connecting to the client: %v", err)
 	}
 	if *manageRoutes {
-		appMgrParms.RouteClientV1, err = routeclient.New(config)
+		rclient, err := routeclient.New(config)
```

Comment:
Ok, I can hide the options. As of right now, not including the routeVserverAddr will enable pool only mode. Do we want to keep that functionality for Routes?